glimmer-vm.code-workspace

```diff
@@ -175,10 +175,6 @@
     {
       "name": "📦 @types/puppeteer-chromium-resolver",
       "path": "packages/@types/puppeteer-chromium-resolver"
-    },
-    {
-      "name": "📦 @types/qunit",
```

Comment:
this whole package was added for one additional property -- it was causing issues on `master` so it was removed